src/index.html

```diff
@@ -11,6 +11,7 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <script> src="https://cdn.ckeditor.com/4.5.11/full/ckeditor.js"></script>
```

Comment:
Por un lado, la etiqueta está rota...
```suggestion
  <script src="https://cdn.ckeditor.com/4.5.11/full/ckeditor.js"></script>
```
Por otro, si quitamos ckeditor, mas bien esta línea habría que eliminarla...